compiler/include/iterator.h

```diff
@@ -73,6 +73,16 @@ void showIteratorGroup(BaseAST* ast);
 void showIteratorGroup(int id);
 
 CallExpr* isSingleLoopIterator(FnSymbol* fn, Vec<BaseAST*>& asts);
+Type*      icMoreFieldType();
+extern int icMoreFieldCode;
+CondStmt*  isIBBCondStmt(BaseAST* ast);
+void       createIteratorBreakBlocks();
+void       addIteratorBreakBlocksInline(Expr* loopRef, Symbol* IC,
+                                        BlockStmt* loopBody, CallExpr* yield,
+                                        std::vector<Expr*>* delayedRemoval);
+BlockStmt* getBreakBlockForYield(std::vector<Expr*>* delayedRemoval,
```

Comment:
Needs a comment about what is being removed (or added to delayed removal). Probably function name should also indicate it removes IBBs